css/types/color.json

```diff
@@ -790,6 +790,106 @@
             }
           }
         },
+        "oklab": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color_value/oklab",
+            "spec_url": "https://drafts.csswg.org/css-color/#ok-lab",
+            "description": "<code>oklab()</code> (Oklab color model)",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": "15.4"
+              },
+              "safari_ios": {
+                "version_added": "15.4"
+              },
```

Comment:
```suggestion
              "safari": {
                "version_added": "preview"
              },
              "safari_ios": {
                "version_added": "preview"
              },
```

Reply:
@queengooborg the screenshot and 137 release notices is enough to show that 15.4 supports `oklch()`?